packages/plugin/src/ContextualMenu.js

```diff
@@ -59,56 +59,54 @@ export const ContextualMenu = ({ config, ouLevels, onClick }) => {
         minWidth: 200,
     }
 
-    return (
+    return ouData ? (
```

Comment:
This is fine for now.
I think originally it was different because we would have other drill options in the menu, like period.
This will need to change anyway when we implement period drilling.